ts/state/selectors/search.ts

```diff
@@ -54,6 +54,23 @@ export const getIsSearchingInAConversation = createSelector(
   Boolean
 );
 
+export const getGlobalSearchValue = createSelector(
+  getSearch,
+  (state: SearchStateType): boolean | undefined => state.globalSearch
+);
+
+export const getIsSearchingGlobally = createSelector(
```

Comment:
small nitpick but we probably don't need the above function we can just coerce into a Boolean right here.

Reply:
Makes sense. Updated!